core/services/vrf/integration_v2_test.go

```diff
@@ -1591,7 +1591,7 @@ func TestIntegrationVRFV2(t *testing.T) {
 	// which should be fixed in this test.
 	ga, err := carolContract.SGasAvailable(nil)
 	require.NoError(t, err)
-	gaDecoding := big.NewInt(0).Add(ga, big.NewInt(3679))
+	gaDecoding := big.NewInt(0).Add(ga, big.NewInt(3701))
```

Comment:
I don't understand why there was a regression here - any ideas @vreff @jinhoonbang ?

Reply:
I only see function name changes in this PR. Does that have any bearing on gas?

Reply:
From my understanding it doesn't - that's also the request flow rather than the response flow, which is unchanged, so I'm not really sure why this is happening.

Reply:
I'd check the failing CI task and see if it is somehow related.

Reply:
It doesn't look like it, it's `TestIntegration_OCR2DR_MultipleRequests_Success` that's failing, seems to be flakey.